pwndbg/color/lexer.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import re
+
+import six
+from pygments.lexer import RegexLexer
+from pygments.lexer import include
+from pygments.token import Comment
+from pygments.token import Name
+from pygments.token import Number
+from pygments.token import Operator
+from pygments.token import Other
+from pygments.token import Punctuation
+from pygments.token import String
+from pygments.token import Text
+
+__all__ = ['PwntoolsLexer']
+
+# Text        Token.Text            for any type of text data
```

Comment:
What exactly are those columns?

Reply:
It is noted in [pygments doc](http://pygments.org/docs/tokens/) and it does not have header row. I just note them and for me to refer easily when editing. I think I can remove this notes.

Reply:
Removed